src/nodes/TSL.js

```diff
@@ -19,6 +19,7 @@ export * from './core/OutputStructNode.js';
 export * from './core/MRTNode.js';
 
 // math
+export * from './math/BitcastNode.js';
```

Comment:
Let's add an export for Nodes too, not just TSL
https://github.com/mrdoob/three.js/blob/dev/src/nodes/Nodes.js